packages/eslint-config-airbnb-base/rules/style.js

```diff
@@ -3,531 +3,823 @@ module.exports = {
+    // DISABLE, Prettier will handle this
     'no-tabs': 'error',
 
     // disallow the use of ternary operators
+    // KEEP OFF
     'no-ternary': 'off',
 
     // disallow trailing whitespace at the end of lines
-    'no-trailing-spaces': ['error', {
-      skipBlankLines: false,
-      ignoreComments: false,
-    }],
+    // KEEP
+    'no-trailing-spaces': [
+      'error',
+      {
+        skipBlankLines: false,
+        ignoreComments: false,
+      },
+    ],
 
     // disallow dangling underscores in identifiers
     // https://eslint.org/docs/rules/no-underscore-dangle
-    'no-underscore-dangle': ['error', {
-      allow: [],
-      allowAfterThis: false,
-      allowAfterSuper: false,
-      enforceInMethodNames: true,
-    }],
+    // KEEP
+    'no-underscore-dangle': [
+      'error',
+      {
+        allow: [],
+        allowAfterThis: false,
+        allowAfterSuper: false,
+        enforceInMethodNames: true,
+      },
+    ],
 
     // disallow the use of Boolean literals in conditional expressions
     // also, prefer `a || b` over `a ? a : b`
     // https://eslint.org/docs/rules/no-unneeded-ternary
+    // KEEP
     'no-unneeded-ternary': ['error', { defaultAssignment: false }],
 
     // disallow whitespace before properties
     // https://eslint.org/docs/rules/no-whitespace-before-property
+    // KEEP
     'no-whitespace-before-property': 'error',
 
     // enforce the location of single-line statements
     // https://eslint.org/docs/rules/nonblock-statement-body-position
+    // SITUATIONAL ANALYSIS NEEDED: Need team input on what they prefer
     'nonblock-statement-body-position': ['error', 'beside', { overrides: {} }],
 
     // require padding inside curly braces
+    // KEEP, this necessitates further discussion. Per Mike's comments on the ES6 style guide, this rule is going to cover the standard
+    // for not only object expressions, but also for reproducing our TypeScript Hero standard for import formatting.
+    // I'd like to cover that in a separate pull request
     'object-curly-spacing': ['error', 'always'],
 
     // enforce line breaks between braces
     // https://eslint.org/docs/rules/object-curly-newline
-    'object-curly-newline': ['error', {
-      ObjectExpression: { minProperties: 4, multiline: true, consistent: true },
-      ObjectPattern: { minProperties: 4, multiline: true, consistent: true },
-      ImportDeclaration: { minProperties: 4, multiline: true, consistent: true },
-      ExportDeclaration: { minProperties: 4, multiline: true, consistent: true },
-    }],
-
-    // enforce "same line" or "multiple line" on object properties.
+    // KEEP, this necessitates further discussion. Per Mike's comments on the ES6 style guide, this rule is going to cover the standard
+    // for not only object expressions, but also for reproducing our TypeScript Hero standard for import formatting.
+    // I'd like to cover that in a separate pull request
+    'object-curly-newline': [
```

Comment:
I'd like to start a discussion here, because if we're going to attempt to replicate TypeScript Hero's import/export formatting, this is where it's going to happen.